code/modules/client/client_procs.dm

```diff
@@ -14,6 +14,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	"1428" = "bug causing right-click menus to show too many verbs that's been fixed in version 1429",
 	"1548" = "bug breaking the \"alpha\" functionality in the game, allowing clients to be able to see things/mobs they should not be able to see.",
 	"1586" = "bug breaking the display of all icons in its entirety, can't be exploited but the game is unplayable on this version.",
+	"1588" = "bug breaking the display of all icons in its entirety, can't be exploited but the game is unplayable on this version."
```

Comment:
```suggestion
	"1588" = "bug breaking the display of all icons in its entirety, can't be exploited but the game is unplayable on this version.",
```
don't forget the comma :3

Reply:
Sir the list is over why would I want the comma.